aries-site/src/pages/components/maskedinput.js

```diff
@@ -23,10 +34,161 @@ const MaskedInput = () => {
             code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/MaskedInputExample/MaskedInputExample.js"
```

Comment:
are we going to add figma and designer links ?

Reply:
Yes! I am going to make designer today and will add the maskedinput link